http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -346,9 +367,21 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
     } yield (stagingContainerName, resp.resourceId)
   }
 
-  private def createDisk(params: CreateAzureRuntimeParams, leoAuth: Authorization)(implicit
+  // Creates a disk and synchronously monitors its creation
+  // This function is written assuming that a disk is created as part of a runtime, and bubbles errors up to the runtime so they are visible to the user
```

Comment:
this assumption is fine for now since we don't have a separate createDisk endpoint for Azure and no Azure apps use disks yet, might have to be adapted later on